portfolio/src/App.tsx

```diff
@@ -1,21 +0,0 @@
-import React, {Component} from 'react';
-import logo from './logo.svg';
-import './App.css';
-import Cover from './components/Cover';
-import Info from './components/Info';
```

Comment:
Prefer named exports in the future, it's easier to import classes and stuff by name with your IDE